DNN Platform/DotNetNuke.ContentSecurityPolicy/ContentSecurityPolicy.cs

```diff
@@ -0,0 +1,476 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.ContentSecurityPolicy
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Manages the entire Content Security Policy.
+    /// </summary>
+    public class ContentSecurityPolicy : IContentSecurityPolicy
+    {
+        private string nonce;
+
+        /// <summary>Initializes a new instance of the <see cref="ContentSecurityPolicy"/> class.</summary>
+        public ContentSecurityPolicy()
+        {
+        }
+
+        /// <summary>
+        /// Gets a cryptographically secure random nonce value for use in CSP policies.
+        /// </summary>
+        public string Nonce
+        {
+            get
+            {
+                if (this.nonce == null)
+                {
+                    var nonceBytes = new byte[32];
+                    var generator = System.Security.Cryptography.RandomNumberGenerator.Create();
```

Comment:
Additionally, RandomNumberGenerator is a class that implements IDisposable, it should at minimum be within a `using` to prevent resource leaking

Reply:
i add using